src/Parser/Druid/Balance/Modules/Features/Abilities.js

```diff
@@ -68,6 +68,26 @@ class Abilities extends CoreAbilities {
       isActive: combatant => !combatant.hasTalent(SPELLS.INCARNATION_CHOSEN_OF_ELUNE_TALENT.id),
       noSuggestion: true,
     },
+    {
+      spell: SPELLS.WARRIOR_OF_ELUNE_TALENT,
+      category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+      getCooldown: (haste, combatant) => {
```

Comment:
This (and Force of Nature) could be written as `getCooldown: haste => 50;` (or 60). If that doesn't work, it's broken elsewhere (I've seen the debate on Discord).